Reference/Security/index-v7.md

```diff
@@ -20,38 +20,31 @@ Don't forget to [configure your Umbraco when using HTTPS](use-https.md).
+The Umbraco backoffice supports external login providers (OAuth) for performing authentication of your users. This could be any OpenIDConnect provider such as Azure Active Directory, Identity Server, Google or Facebook.
 
-The installation of these packages will install snippets of code with readme files on how to get up and running. Depending on the provider you've configured and its caption/color, the end result will look similar to:
+### [BackOfficeUserManager](backoffice-user-manager-v7.3.0.md) and Events
 
-![OAuth login screen](images/google-oauth.png)
+The [`BackOfficeUserManager`](backoffice-user-manager.md) is the ASP.NET Identity [UserManager](https://docs.microsoft.com/en-us/previous-versions/aspnet/dn613290(v=vs.108)) implementation in Umbraco. It exposes APIs for working with Umbraco User's via the ASP.NET Identity including password handling.
 
-#### Auto-linking accounts for custom OAuth providers
-
-Traditionally a backoffice user will need to exist first and then that user can link their user account to an OAuth account in the backoffice. In many cases however, the identity server you choose will be the source of truth for all of your users.
-
-In this case you would want to be able to create user accounts in your identity server and then have that user given access to the backoffice without having to create the user in the backoffice first. This is done via auto-linking.
+### Custom password check
 
-Read more about [auto linking](auto-linking-v7.3.0.md)
+In most cases [External login providers (OAuth)](external-login-providers.md) will meet the needs of most users when needing to authenticate with external resources but in some cases you may need to only change how the username and password credentials are checked.
 
-### Custom password check
+This is typically a legacy approach to validating credentials with external resources but it is possible.
 
-You are able [check the username and password against your own credentials store](custom-password-checker-v7.3.0.md) by implementing a new  `IBackOfficeUserPasswordChecker`.
+You are able check the username and password against your own credentials store by implementing a [`IBackOfficeUserPasswordChecker`](custom-password-checker-v7.3.0.md).
 
-### Authenticating with Active Directory credentials
+#### Authenticating with Active Directory credentials
 
-You want to [connect the backoffice to Active Directory](Authenticate-With-AD-vpre8.md)? Should be pretty straight forward with the `ActiveDirectoryBackOfficeUserPasswordChecker`.
+If you are using a network based Azure Directory (not Azure Active Directory), we have setup a guide on how to [connect the backoffice to Active Directory](Authenticate-With-AD-vpre8.md). It should be pretty straight forward with the `ActiveDirectoryBackOfficeUserPasswordChecker`.
```

Comment:
```suggestion
If you are using a network-based Azure Directory (not Azure Active Directory), we have set up a guide on how to [connect the backoffice to Active Directory](Authenticate-With-AD-vpre8.md). It can be done using the  `ActiveDirectoryBackOfficeUserPasswordChecker`.
```